File: Branches/VBInicial/vpPriV100GrupoMundifios/DsFacade.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Reflection
Imports System.Text
Imports System.Threading.Tasks
Imports Vimaponto.Componentes.Sistema

Public Class DsFacade
    Inherits SisDsServicoDados

    Public Overrides ReadOnly Property NivelBaseDados As Integer
        Get
            Return 2
        End Get
    End Property

    Public Overrides ReadOnly Property AssemblyComScripts As Assembly
        Get
            Return Assembly.GetExecutingAssembly()
        End Get
    End Property

    Public Overrides ReadOnly Property IdentificadorPacote As String
        Get
            Return "GrupoMundifios"
        End Get
    End Property

    Public Sub New(ByRef contexto As SisNeContexto)
        MyBase.New(contexto)

    End Sub
End Class

File: Trunk/vpPriV100Munditalia/DsFacade.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection;
using System.Text;
using System.Threading.Tasks;
using Vimaponto.Componentes.Sistema;

namespace Vimaponto.PrimaveraV100.Clientes.Munditalia
{
    public class DsFacade : SisDsServicoDados
    {
        public override int NivelBaseDados { get { return -1; } }

        public override Assembly AssemblyComScripts { get { return Assembly.GetExecutingAssembly(); } }

        public override string IdentificadorPacote { get { return "Munditalia"; } }

        public DsFacade(ref SisNeContexto contexto) : base(ref contexto)
        {

        }
    }
}
